packages/common/src/api/tan-query/useTopArtists.ts

```diff
@@ -0,0 +1,21 @@
+import { Config } from './types'
+import { useSuggestedArtists } from './useSuggestedArtists'
+import { useTopArtistsInGenre } from './useTopArtistsInGenre'
+
+export const useTopArtists = (genre: string, config?: Config) => {
+  const { data: suggestedArtists, status: suggestedStatus } =
+    useSuggestedArtists({
+      ...config,
+      enabled: genre === 'Featured' && config?.enabled !== false
+    })
+
+  const { data: topArtists, status: topStatus } = useTopArtistsInGenre(
+    { genre },
+    { ...config, enabled: genre !== 'Featured' && config?.enabled !== false }
+  )
+
+  return {
+    data: genre === 'Featured' ? suggestedArtists : topArtists,
+    status: genre === 'Featured' ? suggestedStatus : topStatus
```

Comment:
nit: if it's called Featured, should we just call the hook, slice, etc. featuredArtists instead of suggestedArtists?

Reply:
with conditional logic cant do that sadly

Reply:
oh sry I meant just rename the slice for `suggestedArtists` -> `featuredArtists`

Reply:
but super small potatoes. Just if the genre key is `'Featured'` it made sense to me

Reply:
oh yeah duh misread, sg!

Reply:
Im actually not going to because there is a useFeaturedArtists, whereas suggestedArtists actually uses SUGGESTED_FOLLOWS_HANDLES? Ideally we change it from "featured" to "suggested" in the enum type, but im afraid of breaking things. down to reviisit though when we have all our hooks done and can analyze easier

Reply:
Ahhh yeah nbd